scripts/init_bls_for_stewards

```diff
@@ -0,0 +1,117 @@
+#! /usr/bin/env python3
+
+import argparse
+import logging
+
+from plenum.common.config_util import getConfig
+from plenum.common.constants import BLS_KEY, ALIAS
+from plenum.common.exceptions import OperationError, NoConsensusYet
+from plenum.common.keygen_utils import init_bls_keys
+from plenum.common.signer_did import DidSigner
+from plenum.common.util import hexToFriendly
+from stp_core.common.log import getlogger, Logger
+from stp_core.crypto.nacl_wrappers import Signer
+from stp_core.loop.eventually import eventually
+from stp_core.loop.looper import Looper
+from stp_core.network.port_dispenser import genHa
+from stp_core.types import HA
+
+from indy_client.client.client import Client
+from indy_client.client.wallet.node import Node
+from indy_client.client.wallet.wallet import Wallet
+from indy_common.util import get_reply_if_confirmed
+
+config = getConfig()
+config.enableStdOutLogging = False
+Logger.setLogLevel(logging.INFO)
+logger = getlogger()
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Generate BLS keys for a node "
+                    "by taking the node's name and seeds "
+                    "and send NODE txn with the BLS key specified")
+
+    parser.add_argument('--name', required=True, help='node name')
+    parser.add_argument('--node_seed', required=True, type=str,
```

Comment:
If node_seed is used only for generation of `dest` could you please add option to specify `dest` directly?

Reply:
The requirements for the task is that BLS key needs to be created for a seed (the same seed as used for node keys actually). So I think this option is not required (let's add it if this requirement appears)